GerritDownloader/src/main/java/com/holmsted/gerrit/downloaders/ssh/SshDownloader.java

```diff
@@ -287,6 +287,11 @@ private String createProjectNameList() {
         if (getProjectName().isEmpty()) {
             throw new IllegalStateException("No project name defined!");
         }
-        return String.format("project:^%s", getProjectName());
+        String date = getAfterDate();
+        if (date != null) {
+            return String.format("project:^%s after:%s", getProjectName(), date);
```

Comment:
In which version of Gerrit is 'after' introduced? The tool should at least document that using ---after-date is only available after version 2.X (whatever X is). I checked that at least Gerrit 2.8.4 does not support this, but 2.11 does.